ios/brave-ios/Sources/AIChat/Resources/tr.lproj/BraveLeo.strings

```diff
@@ -229,6 +238,9 @@
 /* Title indicating yearly subscription */
 "aichat.paywallYearlySubscriptionTitle" = "Bir Yıl";
 
+/* The title of the badge where a model which can be used limited */
+"aichat.premiumModelStatusTitle" = "Premium";
```

Comment:
Premium badge will not be translated ?

Reply:
Interesting it's not translated for most languages. Some are still translated.

Reply:
we leave it like this, this may change later